src/main/java/nl/praegus/fitnesse/slim/fixtures/mockserver/MockServer.java

```diff
@@ -265,6 +267,13 @@ private HttpRequest httpRequestMatching(Map<String, Object> rules) {
                         req = req.withHeader(header.getKey().toString(), header.getValue().toString());
                     }
                     break;
+                case "body":
```

Comment:
Moeten we hier nog xpath ondersteuning toevoegen?

Reply:
Ja, dat kunnen we redelijk makkelijk doen. Ik denk wel dat we dan de check moeten aanpassen waarop hij bepaald wat voor soort body het is. Ik denk dat we dan moeten toevoegen "regex=", "jsonpath=" en "xpath=" (en voor de rule er weer vanaf slopen). Want het is lastig een "startWith" te bepalen voor een xpath match die robuust is. Voor jsonpath was dat wel redelijk veilig met de dollar. 

Wat jou betreft ok?

Reply:
Ik denk dat prefixen het meest gebruiksvriendelijk is. De automagische matching i een soort verborgen feature als we dat zouden doen. Overigens is een xpath denk ik altijd één van `//` `./` of `/`? Mar daar zou een regex ook mee kunnen beginnen (of we moeten voor regex de redelijk gebrikelijke `=~` nemen? Dat is dan consistent met de regex functie in slim's check logica.

Ik zie dat je de prefix er al in hebt. Lijkt me prima zo!